libs/platform/unit/platform_unit_test.cpp

```diff
@@ -684,9 +759,14 @@ bitmap_test()
     ebpf_bitmap_t* bitmap = reinterpret_cast<ebpf_bitmap_t*>(data.data());
     ebpf_bitmap_initialize(bitmap, bit_count);
 
-    // Set every even bit interlocked.
-    for (size_t i = 0; i < bit_count; i += 2) {
-        ebpf_bitmap_set_bit(bitmap, i, true);
+    // Set every bit.
```

Comment:
why this change?

Reply:
This is to cover the set and reset interlocked and non-interlocked bitmap operations.